src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/dummy/DummyPlatform.java

```diff
@@ -75,11 +71,7 @@ public String getName() {
      */
     @Override
     public String getID() {
-        String appendix = "";
-        if (name != null) {
-            appendix = name.toLowerCase();
-        }
-        return ID+appendix;
+        return (ID + " " + name).toLowerCase().replaceAll("[^a-z0-9]", "_");
```

Comment:
yes that would be my prefered solution, but with a name like getIdLegal or rename the interfacefunction to getIDRaw so everyone knows that this isnt a complet legal id.